tests/testsuite/crystal.rs

```diff
@@ -0,0 +1,54 @@
+use ansi_term::Color;
```

Comment:
Just a heads-up: these tests aren't actually running on `cargo test` or in the integration script because you didn't add it to `test/testsuite/main.rs`.

But I think if you do that, you'll also have to modify the docker container/test environment to include an install of crystal.

Reply:
I think I'll just leave it off for now. I don't wanna mess with the Dockerfile, I'll just leave that up to @matchai